### PR TITLE
feat: update `depositTo` to use standard deposit

### PR DESCRIPTION
**Pull request by brtkx**
closes FS-368

### CODE REVIEW

File: src/lib/assetBridger/ethBridger.ts [hunk 1]
```diff
@@ -304,58 +295,57 @@ export class EthBridger extends AssetBridger<
...
+        excessFeeRefundAddress: params.destinationAddress,
+        callValueRefundAddress: params.destinationAddress,
+        data: '0x',
+      },
+      await getBaseFee(params.l1Provider),
+      params.l1Provider,
+      params.retryableGasOverrides
+    )
+
+    const tx = await inbox.connect(params.l1Signer).unsafeCreateRetryableTicket(
```
Comment (gzeoneth) [inline src/lib/assetBridger/ethBridger.ts:335]: why are we using `unsafeCreateRetryableTicket` here? can be dangerous

**Review by gzeoneth — CHANGES_REQUESTED** [review]

**Review by gzeoneth — APPROVED** [review]
LGTM

File: src/lib/assetBridger/ethBridger.ts [hunk 2]
```diff
@@ -344,18 +304,75 @@ export class EthBridger extends AssetBridger<
...
+      amount, // l2CallValue
+      gasEstimation.maxSubmissionCost, // maxSubmissionCost
+      destinationAddress, // excessFeeRefundAddress
+      destinationAddress, // callValueRefundAddress
+      gasEstimation.gasLimit, // maxLimit
+      gasEstimation.maxFeePerGas, // maxFeePerGas
+      '0x', // data,
+      {
+        from: signerAddress.value,
+        value: amount,
```
Comment (gzeoneth) [inline src/lib/assetBridger/ethBridger.ts:373]: actually this should be
```suggestion
        value: gasEstimation.deposit,
```

**Review by gzeoneth — CHANGES_REQUESTED** [review]

**Review by gzeoneth — APPROVED** [review]
LGTM

File: tests/integration/eth.test.ts [hunk 3]
```diff
@@ -211,6 +216,60 @@ describe('Ether', async () => {
...
+        gasLimit: {
+          // causes auto-redeem to fail which allows us to check balances before it happens
+          base: constants.Zero,
+        },
+      },
+    })
+    const rec = await res.wait()
+    const l1ToL2Messages = await rec.getL1ToL2Messages(l2Signer.provider!)
+    expect(l1ToL2Messages.length).to.eq(1, 'failed to find 1 l1 to l2 message')
+    let l1ToL2Message = l1ToL2Messages[0]
```
Comment (spsjvc) [inline tests/integration/eth.test.ts:241]:
```suggestion
    const l1ToL2MessageReader = l1ToL2Messages[0]
```

File: tests/integration/eth.test.ts [hunk 4]
```diff
@@ -211,6 +216,60 @@ describe('Ether', async () => {
...
+    ).to.be.true
+
+    const l1TxHash = await l1Signer.provider!.getTransactionReceipt(res.hash)
+    const l1Receipt = new L1TransactionReceipt(l1TxHash)
+    l1ToL2Message = l1Receipt.getL1ToL2Messages(l2Signer)[0]
+
+    if (l1ToL2Message instanceof L1ToL2MessageWriter) {
+      const res = await l1ToL2Message.redeem()
+      await res.wait()
+    }
```
Comment (spsjvc) [inline tests/integration/eth.test.ts:265]:
```suggestion
    const l1ToL2MessageWriter = l1Receipt.getL1ToL2Messages(l2Signer)[0]

    await (await l1ToL2MessageWriter.redeem()).wait()
```

**Review by spsjvc — APPROVED** [review]
LGTM

**Pull request merged**